setup.py

```diff
@@ -19,7 +19,9 @@
     python_requires=">3.5",
     include_package_data=True,
     install_requires=[
-        'itsxpress>=1.7.0'
+        'itsxpress>=1.7.0',
+        'qiime2',
+        'pandas',
```

Comment:
Technically not necessary, but, `pandas` is a secondary dependency of (via `qiime2`), and it buys us some conveniences when it comes to `.view()`.